recipes-security/refpolicy/refpolicy-aos_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "SELinux Aos policy based on targeted policy"
+DESCRIPTION = "\
+This is the Aos modification for targeted variant of the \
+SELinux reference policy. Most service domains are locked \
+down. Users and admins will login in with unconfined_t \
+domain, so they have the same access to the system as if \
+SELinux was not enabled. \
+"
+
+include recipes-security/refpolicy/refpolicy_common.inc
+
+PV = "2.20210908+git${SRCPV}"
+
+SRC_URI = "git://github.com/aoscloud/refpolicy.git;protocol=https;branch=develop;name=refpolicy;destsuffix=refpolicy"
```

Comment:
Add `BRANCH` variable and set SRCREV to ${AUTOREV}. Use same pattern as for other Aos components.

BRANCH = "develop"
SRCREV = "${AUTOREV}"